service/build.gradle

```diff
@@ -51,7 +51,7 @@ jacoco {
 
 sonar {
     properties {
-        property 'sonar.projectName', 'terra-workspace-manager'
+        property 'sonar.projectName', "${rootProject.name}_${projectDir.name}"
```

Comment:
I'm not sure if this is helping, though I did get a coverage report with 98% coverage after I made this change (which was higher than I had gotten previously). Unfortunately it still seems that only the last report shows on the PR, when really we would want to show one per project.